### PR TITLE
Output raw data when piped

### PR DESCRIPTION
**Pull request by daviddavis**
[noissue]

### CODE REVIEW

**Comment by mdellweg** [comment]
Why don't we just switch of the coloring on a tty?
I might want to pipe yaml to a tool that understands it.

**Comment by daviddavis** [comment]
The only caveat is that we are also formatting the json but I removed that too by setting indent to `None` if `isatty`. Also, I added an `isatty` propery.

File: tests/scripts/test_output.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+# shellcheck source=tests/scripts/config.source
+. "$(dirname "$(realpath "$0")")/config.source"
+
+cleanup() {
+  pulp file repository destroy --name "cli_test_file_repo" || true
+}
+trap cleanup EXIT
+
+expect_succ pulp file repository create --name "cli_test_file_repo" --description "Test repository for CLI tests"
```
Comment (mdellweg) [inline tests/scripts/test_output.sh:11]: Is this handled in a way that $OUTPUT looks like it was on a TTY?

Comment (daviddavis) [reply tests/scripts/test_output.sh:11]: Nope, let me try something different.

Comment (mdellweg) [reply tests/scripts/test_output.sh]: So you found something?
I'd be happy to take it without a test, because faking a user session is hard...

Comment (daviddavis) [reply tests/scripts/test_output.sh]: I did. Check it out.

**Review by mdellweg — APPROVED** [review]

**Pull request merged**